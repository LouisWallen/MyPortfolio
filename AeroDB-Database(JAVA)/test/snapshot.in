snapshot
set a 1 2 3
snapshot
set b 7 8
snapshot
list snapshots
rollback 2
checkout 1
drop 1
drop 2
snapshot
snapshot
drop 2
snapshot
snapshot
drop 2
