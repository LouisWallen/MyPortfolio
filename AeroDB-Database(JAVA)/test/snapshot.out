> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> 3
2
1

> ok

> ok

> ok

> ok

> saved as snapshot 1

> saved as snapshot 2

> ok

> saved as snapshot 2

> saved as snapshot 3

> ok
